src/Patterns.php

```diff
@@ -1,7 +1,5 @@
 <?php namespace LeanNs;
```

Comment:
Could we just remove this file completely? I don't think it's required without patternlab